requirements.txt

```diff
@@ -15,6 +15,6 @@ gunicorn==19.6.0
 # c2corg_common project
 # for development use a local checkout
 # -e ../v6_common
-git+https://github.com/c2corg/v6_common.git@e68146023b98acbc312fbba521089a6d2c0651df
+git+https://github.com/c2corg/v6_common.git@d00815d7b961af287b355db38537cbb51a34100a
```

Comment:
Not a big deal but "update req.txt" is not a very explicit commit message, it should rather be "upgrade the v6_common  package" or maybe "update the document listing attributes config"?